app/services/invoices/finalize_service.rb

```diff
@@ -13,7 +13,7 @@ def initialize(invoice:)
 
     def call
       ActiveRecord::Base.transaction do
-        result = Invoices::RefreshDraftService.call(invoice: invoice)
+        result = Invoices::RefreshDraftService.call(invoice:)
```

Comment:
IMO we should check the result from the sub-service here and return error result if any